doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -1,103 +1,136 @@
-# Secure, protect, and connect APIs with Kuadrant on OpenShift
+# Secure, protect, and connect APIs with Kuadrant
 
 ## Overview
 
-This guide walks you through using Kuadrant on OpenShift to secure, protect, and connect an API exposed by a Gateway that is based on Kubernetes Gateway API. You can use this walkthrough for a Gateway deployed on a single OpenShift cluster or a Gateway distributed across multiple OpenShift clusters with a shared listener hostname. This guide shows how the platform engineer and application developer user roles can each use Kuadrant to achieve their goals.
+This guide walks you through using Kuadrant to secure, protect, and connect an API exposed by a Gateway (Kubernetes Gateway API) from the personas platform engineer and application developer. For more information on the different personas please see the [Gateway API documentation](https://gateway-api.sigs.k8s.io/concepts/roles-and-personas/#key-roles-and-personas)
 
-### What Kuadrant can do for you in a multicluster environment
-
-You can leverage Kuadrant's capabilities in single or multiple clusters. The following features are designed to work across multiple clusters as well as in a single-cluster environment.
-
-- **Multicluster ingress:** Kuadrant provides multicluster ingress connectivity using DNS to bring traffic to your Gateways by using a strategy defined in a `DNSPolicy`. 
-- **Global rate limiting:** Kuadrant can enable global rate limiting use cases when configured to use a shared Redis store for counters based on limits defined by a `RateLimitPolicy`.
-- **Global auth:** You can configure a Kuadrant `AuthPolicy` to leverage external auth providers to ensure that different clusters exposing the same API authenticate and authorize in the same way. 
-- **Integration with federated metrics stores:** Kuadrant has example dashboards and metrics for visualizing your Gateways and observing traffic hitting those Gateways across multiple clusters. 
+## Prerequisites
 
-### User roles
+- Completed the steps in [Install Kuadrant on an OpenShift cluster](../../install/install-openshift.md).
```

Comment:
Are we keeping the OpenShift and K8s separate still?  Perhaps should link both options
```suggestion
- Completed the steps in [Install Kuadrant on an OpenShift cluster](../../install/install-openshift.md).
```

Reply:
i had them both and then i ended up separating so it would only have to be tested on openshift for now and get something out for Stephen and then look at merging after